Makefile

```diff
@@ -14,10 +14,18 @@
 
 # TODO(wgrzelak): Invoke targets into the container.
 
-python-test:
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## Shows help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Wow! It works I guess :)

Consider something more explicit than `##`, like `# (--help_text--): Runs tests for Python scripts`

Reply:
I like it, however it make the regex more complex. I'll stay as it is for now.